### ISSUE TITLE
Can't Build APK 

### ISSUE THREAD

**Issue by nadhim1991** [inspireui/support#4561]
Hello,

whenever i try to build APK via following code i get an error.

`

```
flutter build apk --split-per-abi`

`Microsoft Windows [Version 10.0.19041.421]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\src\flutter\fluxStore\FluxStore-vendor-1.7.3\source>flutter build apk --split-per-abi
Codepoint 58848 not found in font, aborting.



FAILURE: Build failed with an exception.

* Where:
Script 'C:\src\flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 896

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command 'C:\src\flutter\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 35s
Running Gradle task 'assembleRelease'...
Running Gradle task 'assembleRelease'... Done                      37.1s
Gradle task assembleRelease failed with exit code 1
`
```

````
Microsoft Windows [Version 10.0.19041.421]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\src\flutter\fluxStore\FluxStore-vendor-1.7.3\source>flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 1.20.1, on Microsoft Windows [Version 10.0.19041.421], locale en-US)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[√] Android Studio (version 4.0)
[√] Connected device (1 available)

• No issues found!

C:\src\flutter\fluxStore\FluxStore-vendor-1.7.3\source>flutter doctor -v
[√] Flutter (Channel stable, 1.20.1, on Microsoft Windows [Version 10.0.19041.421], locale en-US)
    • Flutter version 1.20.1 at C:\src\flutter\flutter
    • Framework revision 2ae34518b8 (5 days ago), 2020-08-05 19:53:19 -0700
    • Engine revision c8e3b94853
    • Dart version 2.9.0


[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\Users\nadhim\AppData\Local\Android\sdk
    • Platform android-R, build-tools 29.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • SM M205F (mobile) • 320156dc623f56ad • android-arm64 • Android 10 (API 29)

• No issues found!

C:\src\flutter\fluxStore\FluxStore-vendor-1.7.3\source>flutter doctor -v
[√] Flutter (Channel stable, 1.20.1, on Microsoft Windows [Version 10.0.19041.421], locale en-US)
    • Flutter version 1.20.1 at C:\src\flutter\flutter
    • Framework revision 2ae34518b8 (5 days ago), 2020-08-05 19:53:19 -0700
    • Engine revision c8e3b94853
    • Dart version 2.9.0


[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\Users\nadhim\AppData\Local\Android\sdk
    • Platform android-R, build-tools 29.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • SM M205F (mobile) • 320156dc623f56ad • android-arm64 • Android 10 (API 29)

• No issues found!

`
```





Thanks

**Comment by skalumek** [comment]
Am getting a similar error

**Comment by ahmadhassan7** [comment]
Can you guys tell me how to connect it to wordpress site. I don't have the files of the wordpress site I only have its url. 
The documentation is poor it tells nothing.
Plus for your solution try cleaning the build first then ```flutter pub get``` and then```flutter build apk --release -vv ```

**Comment by minhcasi** [comment]
@nadhim1991 - pls help to follow our solution on this ticket #4560 

This issue could be related to the latest Flutter `1.20.x`

**Comment by amar0011** [comment]
Hi, I am having the similar error.  Launching lib\main.dart on Android SDK built for x86 in debug mode...
Running Gradle task 'assembleDebug'...
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:88:26: Error: Type 'DiagnosticableMixin' not found.
class PictureStream with DiagnosticableMixin {
                         ^^^^^^^^^^^^^^^^^^^
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:192:44: Error: Type 'DiagnosticableMixin' not found.
abstract class PictureStreamCompleter with DiagnosticableMixin {
                                           ^^^^^^^^^^^^^^^^^^^
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:88:7: Error: The type 'DiagnosticableMixin' can't be mixed in.
class PictureStream with DiagnosticableMixin {
      ^
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:192:16: Error: The type 'DiagnosticableMixin' can't be mixed in.
abstract class PictureStreamCompleter with DiagnosticableMixin {
               ^
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:167:11: Error: Superclass has no method named 'debugFillProperties'.
    super.debugFillProperties(properties);
          ^^^^^^^^^^^^^^^^^^^
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:171:30: Error: The method 'toStringShort' isn't defined for the class 'PictureStreamCompleter'.
 - 'PictureStreamCompleter' is from 'package:flutter_svg/src/picture_stream.dart' ('../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart').
Try correcting the name to the name of an existing method, or defining a method named 'toStringShort'.
      ifPresent: _completer?.toStringShort(),
                             ^^^^^^^^^^^^^
../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:266:11: Error: Superclass has no method named 'debugFillProperties'.
    super.debugFillProperties(description);
          ^^^^^^^^^^^^^^^^^^^


FAILURE: Build failed with an exception.

* Where:
Script 'C:\Program Files\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 896

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\Program Files\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.


Please help. My reg email id is amar.kshk@gmail.com. Purchases a while ago.

**Comment by joemark0008** [comment]
Got same Error

Codepoint 58848 not found in font, aborting.



FAILURE: Build failed with an exception.

* Where:
Script 'C:\src\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 896

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command 'C:\src\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights. 

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 23s
Running Gradle task 'assembleRelease'...
Running Gradle task 'assembleRelease'... Done                      83.9s
Gradle task assembleRelease failed with exit code 1

**Comment by joemark0008** [comment]
> Hi, I am having the similar error. Launching lib\main.dart on Android SDK built for x86 in debug mode...
> Running Gradle task 'assembleDebug'...
> ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:88:26: Error: Type 'DiagnosticableMixin' not found.
> class PictureStream with DiagnosticableMixin {
> ^^^^^^^^^^^^^^^^^^^
> ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:192:44: Error: Type 'DiagnosticableMixin' not found.
> abstract class PictureStreamCompleter with DiagnosticableMixin {
> ^^^^^^^^^^^^^^^^^^^
> ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:88:7: Error: The type 'DiagnosticableMixin' can't be mixed in.
> class PictureStream with DiagnosticableMixin {
> ^
> ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:192:16: Error: The type 'DiagnosticableMixin' can't be mixed in.
> abstract class PictureStreamCompleter with DiagnosticableMixin {
> ^
> ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:167:11: Error: Superclass has no method named 'debugFillProperties'.
> super.debugFillProperties(properties);
> ^^^^^^^^^^^^^^^^^^^
> ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:171:30: Error: The method 'toStringShort' isn't defined for the class 'PictureStreamCompleter'.
> 
> * 'PictureStreamCompleter' is from 'package:flutter_svg/src/picture_stream.dart' ('../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart').
>   Try correcting the name to the name of an existing method, or defining a method named 'toStringShort'.
>   ifPresent: _completer?.toStringShort(),
>   ^^^^^^^^^^^^^
>   ../../../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_svg-0.17.4/lib/src/picture_stream.dart:266:11: Error: Superclass has no method named 'debugFillProperties'.
>   super.debugFillProperties(description);
>   ^^^^^^^^^^^^^^^^^^^
> 
> FAILURE: Build failed with an exception.
> 
> * Where:
>   Script 'C:\Program Files\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 896
> * What went wrong:
>   Execution failed for task ':app:compileFlutterBuildDebug'.
> 
> > Process 'command 'C:\Program Files\flutter\bin\flutter.bat'' finished with non-zero exit value 1
> 
> * Try:
>   Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
> 
> Please help. My reg email id is [amar.kshk@gmail.com](mailto:amar.kshk@gmail.com). Purchases a while ago.

Try to put this 

dependencies:
  flutter_svg: ^0.17.4
dependency_overrides:
  flutter_svg: ^0.18.0

**Comment by minhtruong315** [comment]
@nadhim1991 We released Fluxstore 1.7.7. it's working for flutter 1.20.1 well. please download Fluxstore 1.7.7 on codeanyon and update MStore Api plugin in the wordpress website as well

**Comment by nadhim1991** [comment]
How to update fluxstore without loosing previous customization?
